src/Antiforgery/src/AntiforgeryMiddleware.cs

```diff
@@ -0,0 +1,130 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Abstractions.Metadata;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Antiforgery;
+
+internal sealed partial class AntiforgeryMiddleware
+{
+    private readonly IAntiforgery _antiforgery;
+    private readonly RequestDelegate _next;
+    private readonly ILogger<AntiforgeryMiddleware> _logger;
+
+    public AntiforgeryMiddleware(IAntiforgery antiforgery, RequestDelegate next, ILogger<AntiforgeryMiddleware> logger)
```

Comment:
Is this middleware important enough to be included in security check in `EndpointMiddleware`?
https://github.com/dotnet/aspnetcore/blob/bc118a35bd662a48881975cfc293ee4c4bada01c/src/Http/Routing/src/EndpointMiddleware.cs#L38-L42